controllers/filters.js

```diff
@@ -114,7 +110,11 @@ module.exports.filterByCategoryAndDate = async (
                 $sort: { payDate: -1 },
             },
         ])
-        let sum = calculateSum(expenses)
+
+        let sum = 0
```

Comment:
No need to change it here, but lodash has [sumBy](https://lodash.com/docs/4.17.15#sumBy) for exactly this purpose.

So this line could be:
```typescript
let sum = sumBy(expenses, "cost")
```